src/renderers/dom/shared/__tests__/ReactDOMServerIntegration-test.js

```diff
@@ -455,6 +455,22 @@ describe('ReactDOMServerIntegration', () => {
         expect(parent.childNodes[1].tagName).toBe('DIV');
         expect(parent.childNodes[2].tagName).toBe('DIV');
       });
+
+      itRenders('emptyish values', async render => {
+        let e = await render(0);
+        expect(e.nodeType).toBe(3);
```

Comment:
Want to send follow up? We actually already have this as a constant in the same file. But literal is used in more than one test.